### ISSUE TITLE
Getting a sense of process

### ISSUE THREAD

**Issue by jacoscaz** [w3c/WebID#44]
_/chair hat on_

Hi all,

After a lot of discussion and a lot of work on putting things together and looking for common ground, I think we have a semblance of a way forward in https://github.com/w3c/WebID/issues/37 . It's now time to discuss process, which will determine how to move past the current 2014 ED .

First of all, we can't merge PRs unless we agree on how PRs should be discussed, reviewed and so on.

For now, we are governed by unspecified lazy consensus. Quoting from the [W3C Process Document](https://www.w3.org/2023/Process-20231103/):

> in which lack of objection after sufficient notice is taken as assent

Even if we were to continue with lazy consensus alone, we should at least be explicit in what we mean by "lack of objection" and "sufficient notice". Nonetheless, as chair I find lazy consensus to be downright exhausting and I would definitely appreciate if we could settle on a more structured process. 

These PRs are an attempt at doing so:

- https://github.com/w3c/WebID/pull/31
- https://github.com/w3c/WebID/pull/30

For those who have not done so, yet, **I kindly ask you to review these PRs and provide feedback**.

I can already hear the inevitable question: how do we merge process-related PRs if we do not agree on how to evaluate PRs in general? Well, I'm afraid that's where I need to step in and do my very best at looking for unanimity first and, if that can't be found, doing my very best at moving things forward by unspecified lazy consensus.

Second, a change can only be made against something. What would that something be? We have different choices to make, here:

- Whether to continue editing the 2014 ED or whether to draft a new 2024 ED
- If working on a new document, whether to continue using HTML with ReSpec or whether to move to Markdown with BikeShed, as discussed in https://github.com/w3c/WebID/issues/35

Given that: 

- the group appears to be converging towards a way forward that would bring breaking changes to the 2014 ED
- though EDs are technically not specs, the 2014 ED has been around for so long that it has sort of become a spec

As chair I think the best way forward would be to freeze the 2014 ED and draft a new 2024 ED, starting with an empty document and adding pieces one by one through different PRs. 

What do you think? If you agree with the above, would you make any change to the folder structure of the repository?

_/Chair hat off_

_Personally_ I would prefer if the new document were to be drafted in Markdown with BikeShed.

**Comment by melvincarvalho** [comment]
> though EDs are technically not specs, the 2014 ED has been around for so long that it has sort of become a spec

+1 it was intended that way.  It could have become a REC with LDP but we decided not to include identity in LDP 

> As chair I think the best way forward would be to freeze the 2014 ED and draft a new 2024 ED, starting with an empty document and adding pieces one by one through different PRs.

+1 I came to the conclusion that a freeze is the only way forward, for now.  And we are also waiting on the Solid WG to start up.

Be aware that CGs are the most informal of the W3C Groups.  They are vehicles in which to move fast and iterate.  Especially for new work.

**Comment by jacoscaz** [comment]
Tagging a few people just to make sure they are aware of this (no pressure intended): @namedgraph @acoburn @woutermont @kidehen @jonassmedegaard

**Comment by woutermont** [comment]
Let's clarify something that seems to be causing some serious confusion: the draft of 2014 is the draft of 2014; we cannot go back in time and change that draft. Therefore, whatever we do will be a draft of 2024. I don't see why that causes such a ruckus.

That being said: +1 to all the above.

Re putting @jacoscaz's process doc, I don't immediately see good reason to put in in this repo (it's a personal doc, not meant to be merged into main). If the general feeling is that it needs to be in here anyhow, I suggest **(A)** putting it in a separate branch, or **(B)** putting it in a folder called `proposals` or `working_documents` or similar.

**Comment by jacoscaz** [comment]
Hi all! I've opened the following PRs:

- #47 , which is an alternative to #30 , for README.md. It's basically #30 without the contentious parts.
- #48 , which is an alternative to #31 , for CONTRIBUTING.md. This (aims to) reflect and formally document the current governance by lazy consensus. The idea is to document how the group works _right now_ before making changes to process.
- #46 , which brings in my personal working draft into this repo as requested.

Looking forward to your feedback!

**Comment by jacoscaz** [comment]
Pending more feedback on #49 to see whether it makes sense to further specify our use of lazy consensus, merging in #47 and #48 has made our process explicit and clear, both for our own good and for newcomers. 

Also, it seems like the group is willing to give BikeShed a try and @woutermont has already converted the working draft in #45 , leaving the 2014 ED frozen and untouched in https://github.com/w3c/WebID/tree/main/spec/drafts/ED-webid-20140305 . 

Closing this issue as completed.
